### PR TITLE
remove legacyBundleUMD from npm run dev

### PR DESCRIPTION
**Pull request by oshi97**
It should be fine to remove this since
everybody develops off of answer.js as far
as I know (since that was the only option a few week ago)
TEST=manual
- delete answers.js
- see that npm run dev still generates it
- see that my test page still loads

### CODE REVIEW

File: conf/gulp-tasks/library.gulpfile.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ function compileCSS () {
 function watchJS (cb) {
   return watch(['./src/**/*.js'], {
     ignored: './dist/'
-  }, parallel(bundle, legacyBundleIIFE, legacyBundleUMD));
+  }, parallel(bundle, legacyBundleIIFE));
```
Comment (alexisgrow) [inline conf/gulp-tasks/library.gulpfile.js]: can we remove the answer-modern.js `bundle` as well?

**Review by alexisgrow — COMMENTED** [review]
i like this idea!! it looks like this PR was borked in a rebase maybe?

File: conf/gulp-tasks/library.gulpfile.js (hunk 1 above)
Comment (oshi97) [reply conf/gulp-tasks/library.gulpfile.js]: Yes!!! 👌

**Review by alexisgrow — APPROVED** [review]
I have this locally lol, it is amazing!

**Comment by oshi97** [comment]
> I have this locally lol, it is amazing!

Awesome!

**Pull request merged**